runelite-client/src/main/scripts/ChatBuilder.rs2asm

```diff
@@ -216,9 +218,14 @@ LABEL183:
       91: LABEL207
    jump                   LABEL426
 LABEL186:
+   iload                  9
```

Comment:
Most added lines in rs2asm should be commented since it is difficult to read (eg. what is 9?).